src/test/java/co/rsk/federate/btcreleaseclient/cache/PegoutSignedCacheImplTest.java

```diff
@@ -0,0 +1,92 @@
+package co.rsk.federate.btcreleaseclient.cache;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertSame;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import co.rsk.crypto.Keccak256;
+import co.rsk.federate.signing.utils.TestUtils;
+import java.time.Duration;
+import java.time.Instant;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class PegoutSignedCacheImplTest {
+
+  private static final Duration DEFAULT_TTL = Duration.ofMinutes(30);
+  private static final Keccak256 PEGOUT_CREATION_RSK_HASH = TestUtils.createHash(1);
+
+  private final PegoutSignedCacheImpl pegoutSignedCache = new PegoutSignedCacheImpl(DEFAULT_TTL);
+
+  @BeforeEach
+  void setUp() {
+    pegoutSignedCache.getCache().clear();
```

Comment:
This could be achieved by simply recreating PegoutSignedCacheImpl object

Reply:
Agree with @marcos-iov

Reply:
With the new changes not used anymore.